ArduPlane/Log.cpp

```diff
@@ -427,9 +431,11 @@ const struct LogStructure Plane::log_structure[] = {
 // @Field: Thr: Pre-mixer value for throttle output (between -4500 to 4500)
 // @Field: Rudd: Pre-mixer value for rudder output (between -4500 to 4500)
 // @Field: Flap: Pre-mixer value for flaps output (between -4500 to 4500)
-// @Field: SS: Surface movement / airspeed scaling value
+// @Field: SSx: Surface movement / airspeed scaling value for roll rate control
+// @Field: SSy: Surface movement / airspeed scaling value for pitch rate control
+// @Field: SSz: Surface movement / airspeed scaling value for yaw rate control
     { LOG_AETR_MSG, sizeof(log_AETR),
-      "AETR", "Qffffff",  "TimeUS,Ail,Elev,Thr,Rudd,Flap,SS", "s------", "F------" , true },
+      "AETR", "Qffffffff",  "TimeUS,Ail,Elev,Thr,Rudd,Flap,SSx,SSy,SSz", "s--------", "F--------" , true },
```

Comment:
inflow factor (or 2xinflow_factor) would be good in the log for easy analysis

Reply:
Done